examples/window_resize_increments.rs

```diff
@@ -0,0 +1,68 @@
+#[cfg(any(target_os = "macos", all(target_os = "linux", feature = "x11")))]
+fn main() {
+    use simple_logger::SimpleLogger;
+    use winit::{
+        dpi::LogicalSize,
+        event::{ElementState, Event, KeyboardInput, VirtualKeyCode, WindowEvent},
+        event_loop::EventLoop,
+        window::WindowBuilder,
+    };
+
+    #[cfg(target_os = "macos")]
+    use winit::platform::macos::WindowBuilderExtMacOS;
+
+    #[cfg(all(target_os = "linux", feature = "x11"))]
+    use winit::platform::unix::WindowBuilderExtUnix;
+
+    SimpleLogger::new().init().unwrap();
+    let event_loop = EventLoop::new();
+
+    let window = WindowBuilder::new()
+        .with_title("A fantastic window!")
+        .with_inner_size(LogicalSize::new(128.0, 128.0))
+        .with_resize_increments(LogicalSize::new(25.0, 25.0))
+        .build(&event_loop)
+        .unwrap();
+
+    let mut has_increments = true;
```

Comment:
Given that we have an API for accessing the resize increments, perhaps we could use that instead of the user needing to store a value here?